patches/ios-chrome-browser-shared-model-prefs-browser_prefs.mm.patch

```diff
@@ -18,7 +18,15 @@ index 39a7138041acb7b31ea74a6bf34ec846c526859f..d24fb5d5763129d8d8fe4f1f72483f08
  }
  
  // This method should be periodically pruned of year+ old migrations.
-@@ -710,6 +712,7 @@ void MigrateObsoleteBrowserStatePrefs(PrefService* prefs) {
+@@ -628,6 +630,7 @@ void MigrateObsoleteLocalStatePrefs(PrefService* prefs) {
+                      base::Time::FromNSDate(value));
+     }
+   }
++  BRAVE_MIGRATE_OBSOLETE_LOCAL_STATE_PREFS
```

Comment:
this can be done with a chromium_src override for `MigrateObsoleteLocalStatePrefs`

Reply:
Since this is how we've managed the overrides for the other 3 methods can we keep this for now and then I'll open a new PR removing the patch entirely after this merges?

Reply:
I think it wasn't possible in the other cases because the method name wasn't unique in the file. This is a very simple change to a chromium_src override

Reply:
Yeah I just double checked, can only be done for `MigrateObsoleteLocalStatePrefs` and `MigrateObsoleteBrowserStatePrefs`. I'll update both

Reply:
there's probably still another way to make it work without patches in the other cases, but for now I just want to make sure we're not adding new patches

Reply:
For the time being, replaced the migrate methods over in this PR: https://github.com/brave/brave-core/pull/20264/commits/8a9fb74271757fcc23dd956bf881b5a100ad41e8